charts/prometheus-json-exporter/templates/deployment.yaml

```diff
@@ -42,6 +42,13 @@ spec:
       {{- end }}
       containers:
         - name: {{ .Chart.Name }}
+          {{- if .Values.extraArgs }}
+          args:
+          {{- range .Values.extraArgs }}
+          - {{ .name | quote }}
+          - {{ .value | quote }}
```

Comment:
I don't think this correct.

Reply:
I believe it is. The changes in this PR is currently deployed in one of our environments.